app/models/vocab_sheet.rb

```diff
@@ -1,9 +1,14 @@
 ##
 # A "sheet" of items (signs) saved by a user
 class VocabSheet < ActiveRecord::Base
-  has_many :items
+  has_many :items, dependent: :destroy
 
   def includes_sign?(sign_id)
     items.any? { |i| i.sign_id == sign_id }
   end
+
+  def self.clear_vocab_sheet
+    max_days = 31
```

Comment:
Perhaps max days could be a constant at the top of the model?

MAXIMUM_AGE_IN_DAYS = 31

Reply:
this could be in config. Then we have control of it without changing code and doing a release. We could even turn it off.